composable/assetlist.json

```diff
@@ -356,7 +358,7 @@
         {
           "image_sync": {
             "chain_name": "ethereum",
-            "base_denom": "0x3432B6A60D23Ca0dFCa7761B7ab56459D9C964D0"
+            "base_denom": "0x3432b6a60d23ca0dfca7761b7ab56459d9c964d0"
```

Comment:
The letter casing must match, and I don't see why this is any more (or less) correct than what's currently registered

Reply:
the original submission included upper-case characters, following [etherscan](https://etherscan.io/token/0x3432b6a60d23ca0dfca7761b7ab56459d9c964d0) on this one

Reply:
The thing with etherscan, is when you click "copy", it includes the upper-case characters, but if you manually select the address from the page or url, it will be lowercase only. This is why it's difficult to determine which is correct, even on Etherscan, which is inconsistent.

Reply:
if it must match, then we should go with lowercase. From what I've read online, lowercase is the norm with eth addresses but there can be some cases where upper case letters are found. I normally go with the format in the following section of Etherscan
<img width="459" alt="Screenshot 2024-04-19 at 23 50 40" src="https://github.com/cosmos/chain-registry/assets/107645336/14765a89-97b7-4955-af94-c6c40c9b4f21">

Reply:
In that case the registration for FXS must be updated as well

Reply:
updated